### ISSUE TITLE
Uncaught ReferenceError with scala.js version 1.0.1

### ISSUE THREAD

**Issue by ChristianPolt** [unicredit/akka.js#121]
I updated my project to scala.js version 1.0.1 and got the following runtime exception in my browser console when instantiating an ActorSytem:

```
Uncaught ReferenceError: setImmediate is not defined
    at new $c_Lakka_dispatch_EventLoopExecutor (ExecutorServiceDelegate.scala:14)
    at $p_Lakka_dispatch_Dispatcher$LazyExecutorServiceDelegate__executor$lzycompute__ju_concurrent_ExecutorService (EventLoopExecutorConfigurator.scala:11)
    at $c_Lakka_dispatch_Dispatcher$LazyExecutorServiceDelegate.executor__ju_concurrent_ExecutorService (Dispatcher.scala:41)
    at $c_Lakka_dispatch_Dispatcher.registerForExecution__Lakka_dispatch_Mailbox__Z__Z__Z (ThreadPoolBuilder.scala:69)
    at $c_Lakka_dispatch_Dispatcher.attach__Lakka_actor_ActorCell__V (AbstractDispatcher.scala:156)
    at $f_Lakka_actor_dungeon_Dispatch__start__Lakka_actor_ActorCell (Dispatch.scala:116)
    at $c_Lakka_actor_ActorCell.start__Lakka_actor_ActorCell (ActorCell.scala:408)
    at $c_Lakka_actor_JSLocalActorRefProvider.init__Lakka_actor_ActorSystemImpl__V (ActorRef.scala:349)
    at $c_Lakka_actor_JSLocalActorRefProvider.init__Lakka_actor_ActorSystemImpl__V (JSLocalRefProvider.scala:106)
    at $p_Lakka_actor_ActorSystemImpl__liftedTree2$1__Lakka_actor_ActorSystemImpl (ActorSystem.scala:1035)
```

The following minimal setup reproduces the issue:

build.properties: 
```
sbt.version=1.3.9
```

plugins.sbt:
```
addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.0.1")
```

Main.scala: 
```scala
package core

import akka.actor.ActorSystem

object Main {
  def main(args: Array[String]): Unit = {
    val actorSystem = ActorSystem("System")
    println("This line won't get printed")
  }
}
```

build.sbt:
```sbt
val akkaJsVersion = "2.2.6.3"
lazy val root = (project in file("."))
  .enablePlugins(ScalaJSPlugin)
  .settings(
    name := "akka_js_test",
    scalaVersion := "2.13.1",
    scalaJSUseMainModuleInitializer := true,
    libraryDependencies ++= Seq(
      "org.akka-js" %%% "akkajsactor" % akkaJsVersion
    )
  )
```

and for startup the index.html:
```html
<!DOCTYPE html>
<html>
  <head>
    <script type="text/javascript" src="akka_js_test-fastopt.js"></script>
  </head>
</html>
```

The problem doesn't occur with scala.js version "0.6.32".

**Comment by andreaTP** [comment]
Thanks for the detailed report!
I can reproduce and the underlying issue is most probably:
https://www.scala-js.org/doc/interoperability/global-scope.html
at the section: `Testing whether a global variable exists`

I will fix it soon or feel free to file a PR :-)

**Comment by andreaTP** [comment]
thanks again for reporting!
The fix is available here:
https://github.com/akka-js/akka.js/pull/122
and will be released with 2.2.6.5 as soon as it's available from the Akka side.

**Comment by andreaTP** [comment]
Akka.Js 2.2.6.5 is published on Sonatype!
Thanks for the report @ChristianPolt !
